tests/helpers/fake-redis-client.js

```diff
@@ -11,10 +12,12 @@ module.exports = CoreObject.extend({
   set: function() { },
   del: function() { },
   zadd: function(key, score , tag) {
-    this.recentRevisions.push(key + ':' + tag);
+    assert(key.match(/:revisions$/));
```

Comment:
@achambers @lukemelia I'm doing an assertion for this in the fake client.. it's not super nice and not even needed bug given that we stub redis entirely it seemed the most appropriate place...